docs/library/jesd204/versal_transceiver_subsystem/index.rst

```diff
@@ -0,0 +1,193 @@
+This guide provides instructions on using the wizard to generate a transceiver configuration for JESD204B/C interfaces on Versal devices.
+
+.. note::
+
+   To learn more about Versal ACAP transceivers and the Wizard, please consult:
+
+   * `Versal Adaptive SoC GTY and GTYP Transceivers Architecture Manual (AM002) <https://docs.amd.com/r/en-US/am002-versal-gty-transceivers>`_
+   * `Versal Adaptive SoC Transceiver Subsystem Product Guide (PG442) <https://docs.amd.com/r/en-US/pg442-gtwiz-versal>`_
+
+Versal ACAP Transceiver Architecture
+************************************
+
+Versal ACAP devices feature advanced GTY and GTYP transceivers that support:
+
+- Line rates from 1.2 Gb/s up to 28.21 Gb/s (GTY) and 32.0 Gb/s (GTYP)
+- Multiple PLL options: LCPLL and RPLL for flexible clocking
+- Enhanced performance and power efficiency compared to UltraScale+
+- Native JESD204B and JESD204C protocol support
+
+Required features by the JESD204B
+*********************************
+
+The following features are required for a JESD204B interface:
+
+- LCPLL for clock generation
+- 8B/10B encoding and decoding
+- TX and RX buffer to solve rate and phase differences between XCLK (PMA parallel clock) and USRCLK (link clock)
+- RX Equalization and CDR
+- RX Byte and Word alignment
+
+Required features by the JESD204C
+*********************************
+
+The following features are required for a JESD204C interface:
+
+- LCPLL for clock generation
+- 64B/66B async encoding and decoding
+- TX and RX buffer to solve rate and phase differences between XCLK (PMA parallel clock) and USRCLK (link clock)
+- RX Equalization and CDR
+
+.. _Line Rate and RefClk Selection:
+
+Line Rate and RefClk Selection
+******************************
+
+In our reference designs, we usually use the following relationship between the line rate and the reference clock:
+
+* For JESD204B: Reference clock = Line Rate / 40
+* For JESD204C: Reference clock = Line Rate / 66
+
+Versal Adaptive SoC Transceiver Subsystem wrapper
+*************************************************
+
+In our reference designs, we use a :git-hdl:`versal_xcvr_subsystem.tcl <library/xilinx/scripts/versal_xcvr_subsystem.tcl>`
+as a wrapper to instantiate and configure the Versal Transceiver Subsystem IP core.
+
+The script takes the following parameters:
+
+- ip_name - Name of the IP core instance
+- jesd_mode:
+   - 8b10b - JESD204B
+   - 64b66b - JESD204C
+- rx_no_lanes - Number of RX lanes
+- tx_no_lanes - Number of TX lanes
+- rx_lane_rate - RX lane rate in Gb/s
+- tx_lane_rate - TX lane rate in Gb/s
+- ref_clock - Reference clock frequency in MHz (see `Line Rate and RefClk Selection`_)
+- transceiver - GTY or GTYP
+- intf_cfg:
+   - RX - Simplex RX
+   - TX - Simplex TX
+   - RXTX - Simplex RX + Simplex TX
+
+
+Versal Adaptive SoC Transceiver Subsystem to :ref:`JESD204B/C Link Transmit Peripheral <axi_jesd204_tx>` adapter
```

Comment:
```suggestion
Versal Adaptive SoC Transceiver Subsystem to JESD204B/C Link Transmit Peripheral adapter
```
No need to add link to it since you specify it in the paragraph below